README_JP.md

```diff
@@ -0,0 +1,290 @@
+    if (hTex.IsCreated) {
+        panel[100] = 0;
+        if (hTex.ShouldUpdate(state.Time, 5)) {
+            gClock.BeginGDI();
+            gClock.DrawImage(imgClock, 0, 0);
+            gClock.EndGDI();
+
+            gClock.Graphics.DrawString(
+                (state.Time / 1000).ToString(), 
+                new Font(SystemFonts.DefaultFont.FontFamily, 20, GraphicsUnit.Pixel),
+                Brushes.Black, 0, 0
+            );
+
+            hTex.Update(gClock);
+        }
+    } else {
+     	panel[100] = 1;
+    }
+}
+```
+
+この例では、"clock_back_tex.png" という名前の画像ファイルから作成されたテクスチャへ、DLL と同じディレクトリに存在する "clock_back.png" と現在の日時を表す文字列を描画しています。失敗した場合は運転台パネルで使用可能な値 ats100 を 1 に設定しています。
+
+ここからこのコードの解説をしていきます。
+
+- TextureManager.Register(texturePathSuffix*, width*, *height*);
+
+  シナリオの読込中、`texturePathSuffix` で指定したファイル名の画像からテクスチャが生成されるかどうか監視します。
+  戻り値は TextureHandle 型で、テクスチャが生成された場合は TextureHandle.IsCreated プロパティ (コード例の場合は `hTex.IsCreated`) が true になります。 
+  
+  このメソッドを Load メソッド内で呼び出し、Started メソッド内か Elapse メソッド内で TextureHandle.IsCreated プロパティを確認してください。
+  
+  また、このメソッドは空白のテクスチャで元のテクスチャを置き換えるため注意してください。
+  
+  BVE のマップを車両より先に読み込む仕様の関係で、運転台パネル内のテクスチャは直接置き換えられる一方、マップ内のテクスチャを置き換えるには一度マップを読み込んだ後再度読み込み直す必要があります。
+  そのため、ゲーム開始時に IsCreated が true になっているか確認し、false であれば、コード例のように利用者に対しシナリオを読み込み直す必要がある旨を表示することを推奨します。
+  
+  - texturePathSuffix: 置き換えたいテクスチャのパスの一部。例えば `～～～/Scenarios/shuttle/hrd/structures/back_a.png` を置き換えたい場合は `shuttle/hrd/structures/back_a.png` を指定すると良いでしょう。
+  - width, height: 置換後のテクスチャのサイズ。いずれも 2 のべき乗である必要があります。置換元のテクスチャのサイズと合わせる必要はありません。
+  
+- new GDIHelper(*width*, *height*);
+  テクスチャの描きかえには System.Drawing.Bitmap を使用できますが、Bitmap を直接描きかえる処理の実装は複雑で難易度が高いため、補助を行うための GDIHelper を提供しています。
+
+  GDIHelper のインスタンスはテクスチャ毎に生成してください。
+
+  - GDIHelper.Graphics
+
+    テクスチャに紐づけられた System.Drawing.Graphics を取得します。Graphics を利用すると円や文字列、アルファチャンネル（透過）付画像などを描画することができます。
+  
+    **Graphics.DrawImage メソッドは遅いため推奨しません。GDIHelper.DrawImage メソッドを使用してください。**
+  
+  - GDIHelper.DrawImage(*bitmapToDraw*, *x*, *y*)
+    GDIHelper.DrawImage(*bitmapToDraw*, *x*, *y*, *sourceOffsetY*, *height*)
+  
+    GDI+ による Graphics.DrawImage メソッドは遅いため、GDI32 を利用して Bitmap を描画する代替メソッドを提供しています。
+    ただし、このメソッドの実行前に BeginGDI() メソッド、実行後に EndGDI() メソッドを実行する必要があることに注意してください。
+  
+  - GDIHelper.FillRect12(*color*, *x1*, *y1*, *x2*, *y2*)
+    GDIHelper.FillRectWH(*color*, *x1*, *y1*, *width*, *height*)
+
+    単一色で塗り潰された長方形を描画します。
+    このメソッドの実行前に BeginGDI() メソッド、実行後に EndGDI() メソッドを実行する必要があることに注意してください。
+  
+  BeginGDI() メソッドを実行してから EndGDI メソッドを実行するまでの間に GDIHelper.Graphics を使用しないでください。
+  また、BeginGDI() メソッドを実行してから EndGDI メソッドを実行するまでの間以外で GDIHelper.DrawImage メソッド、GDIHelper.FillRect から始まる名前のメソッドを使用しないでください。
+  
+- TextureHandle.ShouldUpdate(*time*, *fps*)
+
+  `time` で指定した現在時刻を基に、`fps` で指定したフレームレート (テクスチャの更新頻度) の下限を下回っているかを取得します。
+  テクスチャの更新はコストが大きいため、できるだけ頻度を下げることが推奨されます。
+
+  このメソッドは単に「最後の更新からどのくらいの時間が経過したか」を確認しているのみなので、このメソッドを使わず、強制的に更新することもできます。
```

Comment:
I just changed the original English document for this and cancelled the `time` parameter. Please take a look.
I think it might be better to emphasize that "Update" can be called at any time, and `HasEnoughTimePassed` is offering a suggestion to limit the update frequency.

Reply:
Updated in [ba86bc3](https://github.com/zbx1425/DXDynamicTexture/pull/1/commits/ba86bc3ca423cec009ab01a16b66febc8d9abc84) to use `HasEnoughTimePassed` method.